soc/riscv/riscv-ite/CMakeLists.txt

```diff
@@ -1,3 +1,2 @@
 add_subdirectory(common)
 add_subdirectory(${SOC_SERIES})
-zephyr_compile_options(-march=rv32imac)
```

Comment:
I don't see anything in the PR comment that says anything about changing the compile options, I only see the other file's changes mentioned.

Reply:
Done.

Reply:
Still not seeing a change. I edited the comment above to say this was being changed but have no idea why it is needed to enable the A/M/F extensions.